crates/vm/levm/src/vm.rs

```diff
@@ -380,65 +359,166 @@ impl VM {
             ))?
             .clone();
 
+        self.decrement_account_nonce(call_frame.msg_sender)?;
+
+        let new_contract_address = call_frame.to;
+        if cache::remove_account(&mut self.cache, &new_contract_address).is_none() {
+            return Err(VMError::AddressDoesNotMatchAnAccount); // Should not be this error
+        }
+
+        // Should revert this?
+        // sender_account.info.balance -= self.call_frames.first().ok_or(VMError::FatalUnwrap)?.msg_value;
```

Comment:
I know that comment is not yours, but if it is not necessary we can remove it

Reply:
I removed it. I think that in my next PR I'm going to remove `revert_create()` function anyways.
https://github.com/lambdaclass/ethrex/pull/1319/commits/c9b747f7990ea6239daa17cf57bc9f459687b24d